packages/core/src/blockchain/index.ts

```diff
@@ -49,6 +49,11 @@ export interface Options {
   maxMemoryBlockCount?: number
   /** Whether to process queued messages */
   processQueuedMessages?: boolean
+  /**
+   * Storage Proof Size passed to runtime.
+   * See <https://github.com/smol-dot/smoldot/blob/26dedea00eafa7073ca1d4678277272d807b3d89/lib/src/executor/host.rs#L248>
+   */
+  storageProofSize?: number
```

Comment:
no need to put it here. this should be a per call option and defaults to 0 and for now the only way to set it to some value is from using the API directly

Reply:
which api do you refer to here? and how do user pass in this value